share/picongpu/examples/FieldAbsorberTest/include/picongpu/param/fieldBackground.param

```diff
@@ -84,6 +86,39 @@ namespace picongpu
         /* Add this additional field? */
         static constexpr bool activated = true;
 
+        /* This setup is based on [Taflove, Hagness], section 7.11.1.
+         * The difference is we consider both 2D and 3D cases, and grid size may be increased due to our absorber being
+         * part of the simulation area, not located outside of it as in the book.
+         *
+         * Example of a rectangular conductor with a steady current.
+         *
+         * The conductor is oriented along the an axis.
+         * It shall have a diameter of 32 cells.
+         * The current within the wire shall be I = 1A which corresponds to a current density
+         *     j_0 = 1 A / ( 32*32 * SI::CELL_WIDTH_SI * SI::CELL_HEIGHT_SI ).
+         *
+         * The magnetic field circles around the wire and its strength is inversely proportional to the distance s from
+         * the wire
+         *     B = B_0 * (a / s),
+         * where
+         *     B_0 = mu_0 * j_0 * a / 2
+         * is the magnetic field on the surface of the wire.
+         *
+         * The temporal increase of the current I, and therefore its density too, can be regulated similar to an RL
+         * circuit with a coil of inductance L:
+         *     I(t) = I_0 * [1 - e^(- R * t / L)]
+         * https://www.leifiphysik.de/elektrizitaetslehre/elektromagnetische-induktion/ausblick/theorie-zum-einschalten-von-rl-kreisen
```

Comment:
Is there a similar reference in English?

Reply:
I actually don't know, @steindev do you have an idea?